llvm/lib/Analysis/ScalarEvolutionAliasAnalysis.cpp

```diff
@@ -61,6 +61,20 @@ AliasResult SCEVAAResult::alias(const MemoryLocation &LocA,
                                  ? static_cast<uint64_t>(LocB.Size.getValue())
                                  : MemoryLocation::UnknownSize);
 
+    // Firstly, try to convert the two pointers into ptrtoint expressions to
+    // handle two pointers with different pointer bases.
+    // Either both pointers are used with ptrtoint or neither, so we can't end
+    // up with a ptr + int mix (not entirely sure whether it's possible for just
+    // one of the conversions to fail).
```

Comment:
```suggestion
    // up with a ptr + int mix.
```
I don't think this belongs in a code comment :)

Reply:
It has been fixed. Thanks a lot for your suggestion.